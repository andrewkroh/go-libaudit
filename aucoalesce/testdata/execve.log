type=SYSCALL msg=audit(1492037291.036:59): arch=c000003e syscall=59 success=yes exit=0 a0=1522170 a1=1506120 a2=14faeb0 a3=7ffcc4582df0 items=2 ppid=12041 pid=13393 auid=1000 uid=1000 gid=1001 euid=1000 suid=1000 fsuid=1000 egid=1001 sgid=1001 fsgid=1001 tty=pts1 ses=802 comm="grep" exe="/usr/bin/grep" subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 key="user_commands"
type=EXECVE msg=audit(1492037291.036:59): argc=3 a0="grep" a1="--color=auto" a2="mapping"
type=CWD msg=audit(1492037291.036:59):  cwd="/home/andrew_kroh"
type=PATH msg=audit(1492037291.036:59): item=0 name="/usr/bin/grep" inode=2512 dev=08:01 mode=0100755 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:bin_t:s0 objtype=NORMAL
type=PATH msg=audit(1492037291.036:59): item=1 name="/lib64/ld-linux-x86-64.so.2" inode=19161168 dev=08:01 mode=0100755 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:ld_so_t:s0 objtype=NORMAL
type=EOE msg=audit(1492037291.036:59)
