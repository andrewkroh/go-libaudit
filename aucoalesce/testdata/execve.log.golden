{
  "@timestamp": "2017-04-12T22:48:11.036Z",
  "cwd": "/home/andrew_kroh",
  "execve": {
    "argc": "3",
    "cmdline": "\"grep\" \"--color=auto\" \"mapping\""
  },
  "path": [
    {
      "dev": "08:01",
      "inode": "2512",
      "item": "0",
      "mode": "0100755",
      "name": "/usr/bin/grep",
      "obj_domain": "bin_t",
      "obj_level": "s0",
      "obj_role": "object_r",
      "obj_user": "system_u",
      "objtype": "NORMAL",
      "ogid": "0",
      "ouid": "0",
      "rdev": "00:00"
    },
    {
      "dev": "08:01",
      "inode": "19161168",
      "item": "1",
      "mode": "0100755",
      "name": "/lib64/ld-linux-x86-64.so.2",
      "obj_domain": "ld_so_t",
      "obj_level": "s0",
      "obj_role": "object_r",
      "obj_user": "system_u",
      "objtype": "NORMAL",
      "ogid": "0",
      "ouid": "0",
      "rdev": "00:00"
    }
  ],
  "sequence": 59,
  "syscall": {
    "a0": "1522170",
    "a1": "1506120",
    "a2": "14faeb0",
    "a3": "7ffcc4582df0",
    "arch": "x86_64",
    "auid": "1000",
    "comm": "grep",
    "egid": "1001",
    "euid": "1000",
    "exe": "/usr/bin/grep",
    "exit": "0",
    "fsgid": "1001",
    "fsuid": "1000",
    "gid": "1001",
    "key": "user_commands",
    "name": "execve",
    "pid": "13393",
    "ppid": "12041",
    "ses": "802",
    "sgid": "1001",
    "subj_category": "c0.c1023",
    "subj_domain": "unconfined_t",
    "subj_level": "s0-s0",
    "subj_role": "unconfined_r",
    "subj_user": "unconfined_u",
    "success": "yes",
    "suid": "1000",
    "tty": "pts1",
    "uid": "1000"
  }
}
